GNUmakefile

```diff
@@ -50,8 +49,7 @@ testaccgov: fmtcheck
 .PHONY: fmt
 fmt:
 	@echo "==> Fixing source code with gofmt..."
-	gofmt -s -w ./main.go
-	gofmt -s -w ./$(PKG_NAME)
+	gofmt -s -w .
```

Comment:
I suppose we don't need to change pre-commit as it goes file by file

Reply:
Correct. This was detected when running `make build` and seeing the failure in `make fmtcheck`